fastify.js

```diff
@@ -78,6 +78,10 @@ function build (options) {
     started = true
   })
 
+  function throwIfAlreadyBound (msg) {
+    if (listening) throw new Error(msg)
```

Comment:
I think is better to check [`started`](https://github.com/allevo/fastify/blob/25b86cb58c510444dcae7661db73c5f9966c0d7f/fastify.js#L78) variable, in this way we will throw also in case you call `inject`, `ready` and so on :)

Reply:
Truly with `started` the tests fail!

I'm looking for why.